metrics/metrics/src/test/java/io/helidon/metrics/TestServer.java

```diff
@@ -64,7 +65,7 @@ public class TestServer {
     private static MetricsSupport metricsSupport;
 
     private static final Duration CLIENT_TIMEOUT = Duration.ofSeconds(5);
```

Comment:
I had included property look-up of `RETRY_ COUNT` and `RETRY_DELAY_MS` so we would have an easy way to adjust for slow pipelines or action runners.

I know `CLIENT_TIMEOUT` was not originally set up that way, but do you think it is worth doing a similar property look-up for it for the same reason as part of this PR?

Reply:
Right